Code/Samples/ShaderExplorer/ShaderExplorer.cpp

```diff
@@ -244,10 +249,28 @@ void ezShaderExplorerApp::AfterCoreSystemsStartup()
   ezGALDeviceFactory::GetShaderModelAndCompiler(sRendererName, szShaderModel, szShaderCompiler);
 
   ezShaderManager::Configure(szShaderModel, true);
-  EZ_VERIFY(ezPlugin::LoadPlugin(szShaderCompiler).Succeeded(), "Shader compiler '{}' plugin not found", szShaderCompiler);
+  ezPlugin::LoadPlugin(szShaderCompiler).IgnoreResult();
```

Comment:
LoadOptionalPlugin ?

Reply:
Changed it to `ezPlugin::LoadPlugin("ezInspectorPlugin", ezPluginLoadFlags::PluginIsOptional).IgnoreResult();`
Still need to discard the result thought.